### PR TITLE
Fix add group member and change group resource managing entity

### PR DESCRIPTION
**Pull request by ellykits**
Add group member should run after the generated member resources have been created and saved to the DB.

Change managing entity should use the same code to identify the RelatedPerson object.

**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes [link to issue]

**Engineer Checklist**
- [ ] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [ ] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [ ] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [ ] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 
- [ ] I have checked that this PR does NOT introduce **breaking changes** that require an update to **_Content_** and/or **_Configs_**? _If it does add a sample here or a link to exactly what changes need to be made to the content._

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: android/engine/src/main/java/org/smartregister/fhircore/engine/sync/AppSyncWorker.kt [hunk 1]
```diff
@@ -49,7 +49,7 @@ constructor(
 
   /** Disable ETag for upload */
   override fun getUploadConfiguration(): UploadConfiguration =
-    UploadConfiguration(useETagForUpload = false)
+    UploadConfiguration(useETagForUpload = false, uploadBundleSize = 100)
```
[the remark this answers is not in the document]
Comment (pld) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/sync/AppSyncWorker.kt:52]: oh, I missed this was going into 025, for this branch, I'd say leave as is, do not refactor to hard-coded config, but if we want this change in main, refactor at that point

[the remark this answers is not in the document]
Comment (ellykits) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/sync/AppSyncWorker.kt:52]: Oh I missed that. That was included while were were debugging the sync issue. My bad.

[the remark this answers is not in the document]
Comment (ellykits) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/sync/AppSyncWorker.kt:52]: I will revert.

[the remark this answers is not in the document]
Comment (ellykits) [reply android/engine/src/main/java/org/smartregister/fhircore/engine/sync/AppSyncWorker.kt]: @pld Just to let you know the fixes on this PR are already merged in `main` branch.

**Review by dubdabasoduba — APPROVED** [review]

**Review by ageryck — APPROVED** [review]

**Pull request merged**